examples/tutorials/10_Exploring_Quantum_Chemistry_with_GDB1k.ipynb

```diff
@@ -193,6 +194,41 @@
         "The ntext step we want to do is load our dataset. We're using a small dataset we've prepared that's pulled out of the larger GDB benchmarks. The dataset contains the atomization energies for 1K small molecules."
       ]
     },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "zsFfAmAgGQIk",
+        "colab_type": "code",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 208
+        },
+        "outputId": "d818ec76-bc9a-458a-9164-483d63d0cde4"
+      },
+      "source": [
+        "!wget https://raw.githubusercontent.com/deepchem/deepchem/master/datasets/gdb1k.sdf"
```

Comment:
Can you convert this url to `https://deepchemdata.s3-us-west-1.amazonaws.com/datasets/gdb1k.sdf`?
We try to remove datasets in `deepchem/datasets`.